build-tools/packages/version-tools/package.json

```diff
@@ -11,6 +11,13 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"exports": {
+		".": {
+			"types": "./lib/index.d.ts",
+			"import": "./lib/index.js",
+			"require": "./dist/index.js"
+		}
+	},
 	"main": "lib/index.js",
 	"types": "lib/index.d.ts",
```

Comment:
The policy checker should probably verify that the types/main/module fields match the values in exports. Ideally it wouls also check if the files exist, but since they're built files there's no easy way to do that.